x-pack/plugins/security_solution/common/api/entity_analytics/asset_criticality/upload_asset_criticality_csv.schema.yaml

```diff
@@ -0,0 +1,76 @@
+openapi: 3.0.0
+info:
+  version: 1.0.0
+  title: Asset Criticality Create Record Schema
+servers:
+  - url: 'http://{kibana_host}:{port}'
+    variables:
+      kibana_host:
+        default: localhost
+      port:
+        default: '5601'
+paths:
+  /internal/asset_criticality/upload_csv:
+    post:
+      summary: Uploads a CSV file containing asset criticality data
+      requestBody:
+        content:
+          multipart/form-data:
+            schema:
+              type: object
+              properties:
+                file:
+                  type: string
+                  format: binary
+                  description: The CSV file to upload.
+              required:
+                - file
+      responses:
+        '200':
+          description: CSV upload successful
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AssetCriticalityCsvUploadResponse'
+        '403':
```

Comment:
Just confirming that Kibana's router actually throws a 403 in the situation of `maxBytes` being exceeded? Typically, that should be a [413](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/413)

Reply:
Good eye! You're right, fixed in [e66f9d2](https://github.com/elastic/kibana/pull/179930/commits/e66f9d2152d3e609b917717994b57e4fa7c47b24)